kcards/static/options.js

```diff
@@ -0,0 +1,9 @@
+var clipboard = new Clipboard(".btn");
+
+clipboard.on('success', function(e) {
+    $('#flashes').append('<p class="info">Link copied to clipboard.</p>');
```

Comment:
For reference: http://stackoverflow.com/questions/24288380/flask-flash-and-url-for-with-ajax